docs/source/quicktour.mdx

```diff
@@ -211,220 +211,93 @@ and get tensors back. You can specify all of that to the tokenizer:
-
+See the [task summary](./task_summary) for which [`AutoModel`] to use for which task.
 </Tip>
 
-Let's apply the SoftMax activation to get predictions.
+Now you can pass your preprocessed batch of inputs directly to the model. If you are using a PyTorch model, unpack the dictionary by adding `**`. For TensorFlow models, pass the dictionary keys directly to the tensors:
 
-```py
->>> from torch import nn
->>> pt_predictions = nn.functional.softmax(pt_outputs.logits, dim=-1)
+```python
+>>> pt_outputs = pt_model(**pt_batch)
 ===PT-TF-SPLIT===
->>> import tensorflow as tf
->>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
+>>> tf_outputs = tf_model(tf_batch)
 ```
 
-We can see we get the numbers from before:
+The model outputs the model's final activations in the `logits` attribute. Apply the softmax activation to the `logits` to retrieve the predictions:
 
-```py
+```python
+>>> from torch import nn
+>>> pt_predictions = nn.functional.softmax(pt_outputs.logits, dim=-1)
 >>> print(pt_predictions)
 tensor([[2.2043e-04, 9.9978e-01],
         [5.3086e-01, 4.6914e-01]], grad_fn=<SoftmaxBackward>)
 ===PT-TF-SPLIT===
->>> print(tf_predictions)
+>>> import tensorflow as tf
+>>> tf_predictions = tf.nn.softmax(tf_outputs.logits, axis=-1)
+>>> print(tf_predictios)
 tf.Tensor(
 [[2.2043e-04 9.9978e-01]
  [5.3086e-01 4.6914e-01]], shape=(2, 2), dtype=float32)
 ```
 
-If you provide the model with labels in addition to inputs, the model output object will also contain a `loss`
-attribute:
-
-```py
->>> import torch
->>> pt_outputs = pt_model(**pt_batch, labels = torch.tensor([1, 0]))
->>> print(pt_outputs)
-SequenceClassifierOutput(loss=tensor(0.3167, grad_fn=<NllLossBackward>), logits=tensor([[-4.0833,  4.3364],
-        [ 0.0818, -0.0418]], grad_fn=<AddmmBackward>), hidden_states=None, attentions=None)
-===PT-TF-SPLIT===
->>> import tensorflow as tf
->>> tf_outputs = tf_model(tf_batch, labels = tf.constant([1, 0]))
->>> print(tf_outputs)
-TFSequenceClassifierOutput(loss=<tf.Tensor: shape=(2,), dtype=float32, numpy=array([2.2051e-04, 6.3326e-01], dtype=float32)>, logits=<tf.Tensor: shape=(2, 2), dtype=float32, numpy=
-array([[-4.0833 ,  4.3364  ],
-       [ 0.0818, -0.0418]], dtype=float32)>, hidden_states=None, attentions=None)
-```
-
-Models are standard [torch.nn.Module](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) or [tf.keras.Model](https://www.tensorflow.org/api_docs/python/tf/keras/Model) so you can use them in your usual training loop. 🤗 Transformers also provides a [`Trainer`] class to help with your training in PyTorch (taking care of things such as distributed training, mixed precision, etc.) whereas you can leverage the `fit()` method in Keras. See the [training tutorial](training) for more details.
-
 <Tip>
+All 🤗 Transformers models (PyTorch or TensorFlow) return the model activations *before* the final activation
+function (like softmax) because the final activation function is often fused with the loss.
```

Comment:
```suggestion
All 🤗 Transformers models (PyTorch or TensorFlow) outputs the tensors *before* the final activation
function (like softmax) because the final activation function is often fused with the loss.
```